examples/rando/include/rando.h

```diff
@@ -11,16 +11,26 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+#ifndef SNAP_PLUGIN_COLLECTOR_RANDO
+#define SNAP_PLUGIN_COLLECTOR_RANDO
+
+#include<vector>
```

Comment:
You may want to add `cpplint` to the project and don't worry about style: https://pypi.python.org/pypi/cpplint